operator/src/main/java/oracle/kubernetes/weblogic/domain/model/DomainStatus.java

```diff
@@ -88,6 +92,12 @@ public class DomainStatus {
           + "or clusters are available.")
   private OffsetDateTime startTime = SystemClock.now();
 
+  @Description("RFC 3339 date and time at which a currently failing domain started automatic retries.")
```

Comment:
A quibble: I am no expert on time standards by any stretch of the imagination but replacing 'RFC 3339' with 'ISO 8601' may be a little more appropriate. The javadoc for the OffsetDateTime method that's used here directly states that it uses the ISO 8601 standard, and, according to [this article](https://medium.com/easyread/understanding-about-rfc-3339-for-datetime-formatting-in-software-engineering-940aa5d5f68a), the main difference is that it requires the 'T' where-as RFC 3339 also allows using a space instead. We use the same 'T' format for the operator logs and introspector scripts.

Reply:
Hmm... RFC 3339 is what we've been saying with the other time values I found, including DomainStatus.startTime and ServerHealth.activationTime, and [this article](https://stackoverflow.com/questions/522251/whats-the-difference-between-iso-8601-and-rfc-3339-date-formats) disputes the article you cited.

@rjeberhard your call on this.

Reply:
Yes, Kubernetes time-based data types are RFC 3339: https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#metadata

Reply:
Here is how the creationTimestamp in metadata is described:
"CreationTimestamp is a timestamp representing the server time when this object was created. It is not guaranteed to be set in happens-before order across separate operations. Clients may not set this value. It is represented in RFC3339 form and is in UTC. Populated by the system. Read-only. Null for lists. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#metadata
Returns:
creationTimestamp"

Reply:
I was looking at the javadoc for the API, which is pretty clear  `A date-time with an offset from UTC/Greenwich in the ISO-8601 calendar system, such as 2007-12-03T10:15:30+01:00.`.  No worries - the two standards are only slightly different - RFC being a superset.

Reply:
Yes, I think that you're right, Tom. The implementation is using ISO-8601 behavior because it aligns with RFC 3339 when UTC is enforced.